### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by galk-in**
- Move missed MissingEnvVarsError function to index file  (#19)
- Fix inconsistent tests
- Add eslint for code style checks

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```
Comment (rolodato) [inline .editorconfig]: Please update .editorconfig to match comment in .eslintrc.js.

File: CHANGELOG.markdown [hunk 2]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.1] - 2017-01-03
+### Changed
+- Move missed MissingEnvVarsError function to index file
```
Comment (rolodato) [inline CHANGELOG.markdown]: This should be:

```
### Fixed
- Fix missing `MissingEnvVarsError.js` error when calling `load()`

File: CHANGELOG.markdown [hunk 3]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
...
+- Fix inconsistent tests
```
Comment (rolodato) [inline CHANGELOG.markdown]: Not user facing, shouldn't be in changelog.

File: CHANGELOG.markdown [hunk 4]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
...
+- Add eslint for code style checks
```
Comment (rolodato) [inline CHANGELOG.markdown]: Not user facing, shouldn't be in changelog.

File: package.json [hunk 5]
```diff
@@ -1,29 +1,32 @@
...
-    "test": "cd test && HELLO=fromTheOtherSide mocha -CR spec index.js"
-  },
   "keywords": [
     "dotenv"
   ],
+  "license": "MIT",
+  "author": "Rodrigo López Dato <rlopezdato@gmail.com>",
   "files": [
     "index.js",
     "config.js"
```
Comment (rolodato) [inline package.json]: `"MissingEnvVarsError.js"` should be added here

**Comment by galk-in** [comment]
Closed as outdated.

**Pull request closed without merging**